test_settings.py

```diff
@@ -18,10 +18,24 @@ def root(*args):
     return join(abspath(dirname(__file__)), *args)
 
 
+ALLOWED_COMPLETION_AGGREGATOR_EVENT_TYPES = {
+    "progress": {
+        "course",
+        "chapter",
+        "sequential",
+        "vertical",
+    },
+    "completion": {
+        "course",
+        "chapter",
+        "sequential",
+        "vertical",
+    }
+}
```

Comment:
Hi @Agrendalath 

I'm reconsidering these event types.. we're only emitting a "progressed" event for [ordinary completions](https://github.com/openedx/event-routing-backends/blob/master/event_routing_backends/processors/xapi/event_transformers/completion_events.py#L13), with the `completed` flag indicating completion. I'm going to do the same here, and remove these "completion" event types.